sources/vega/events/v1/events.proto

```diff
@@ -12,7 +12,31 @@ import "vega/oracles/v1/data.proto";
 import "vega/commands/v1/commands.proto";
 import "vega/commands/v1/validator_commands.proto";
 
-//DelegationBalanceEvent - updates on the delegation balance of a party to a node in the current epoch in effect 
+// StakingEvent - an event notifying of stake being deposited or removed for a given party
+// These events are emitted for every Staking deposit or removed accepted by the network
+message StakingEvent {
+  enum Kind {
+    // Default value
+    KIND_UNSPECIFIED = 0;
```

Comment:
Is this another one of those kind vs type things, or can we use `Type` instead of `Kind` here?

Reply:
We can if you prefer, I'm not fussy about it.

Reply:
addressed.